src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/manual-remove/ManualRemoveView.tsx

```diff
@@ -2,6 +2,9 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+"use client";
```

Comment:
<img width="862" alt="Screenshot 2023-10-15 at 8 28 40 PM" src="https://github.com/mozilla/blurts-server/assets/42494162/fb753a20-48ec-466b-8196-1e767cd19ca1">
@Vinnl getting this client component error... is the only way to fix this to expose it as a server component?

Reply:
Ah yes, so now that you're using client-side state, this needs to be a client component.

However, Fluent's `ReactLocalization` class is derived from the user's `Accept-Language`, which we need to access in different ways on the client and on the server. So for server components, we use `getL10n`, but if you're turning this into a client component, you'll have to replace that with the `useL10n` React hook, which should fix that error.

Reply:
Oh didn't know we have a hook for this. That makes it easier! Thanks!